doc/examples.rst

```diff
@@ -387,3 +387,126 @@ Flag without argument:
    http://127.0.0.1:1024/tarantool
    {"res":"Tarantool CLI version 0.1.0, darwin/amd64. commit: bf83f33\n
     • Tarantool executable found: '/usr/local/bin/tarantool'\n"}
+
+Transition from tarantoolctl to tt
+----------------------------------
+Suppose we have a set of Tarantool instances managed by ``tarantoolctl`` utility. In order for the tt
+to work with these instances run ``tt init`` command in the directory where ``tarantoolctl``
+configuration file (``.tarantoolctl``) is located. For example:
+
+.. code-block:: bash
+
+    $ cat .tarantoolctl 
+    default_cfg = {
+        pid_file  = "./run/tarantool",
+        wal_dir   = "./lib/tarantool",
+        memtx_dir = "./lib/tarantool",
+        vinyl_dir = "./lib/tarantool",
+        log       = "./log/tarantool",
+        language  = "Lua",
+    }
+    instance_dir = "./instances.enabled"
+
+    $ tt init
+    • Found existing config '.tarantoolctl'
+    • Environment config is written to 'tt.yaml'
+
+``tt init`` takes directories paths from the existing ``tarantoolctl`` config. Generated ``tt.yaml``
+will look like:
+
+.. code-block:: yaml
+
+    tt:
+    modules:
+        directory: modules
+    app:
+        run_dir: ./run/tarantool
+        log_dir: ./log/tarantool
+        log_maxsize: 100
+        log_maxage: 8
+        log_maxbackups: 10
+        restart_on_failure: false
+        data_dir: ./lib/tarantool
```

Comment:
This parameter has been split.

Reply:
The `tarantoolctl_layout` parameter is now also added.

Reply:
Updated.